### PR TITLE
Adds Foam Dart Shotgun as a loot crate drop

### PR DESCRIPTION
**Pull request by HatNJacket**
[game objects]

## About the PR 

This PR adds a foam dart shotgun. It shoots 6 foam darts with a capacity of 12. It is a large item able to be put into backpacks, or slung over a spaceman's shoulder. It has a chance to be within loot crates, which are found in the debris field or by mining. One also can be found in space, among the original foam weapons...

Yahar

ezgif com-gif-maker (11)

## Why's this needed? 

The foam dart gun arsenal is currently lacking larger guns to facilitate maximum foam dart war silliness, a shotgun that fires 6 foam darts at once will fix that lack of harmless firepower.

## Changelog 

```changelog
(u)KolisDumb, DeadMan164
(+)Adds a foam dart shotgun to the loot crate item pool.
```

### CODE REVIEW

File: code/obj/item/gun/kinetic.dm [hunk 1]
```diff
@@ -1034,6 +1034,28 @@ ABSTRACT_TYPE(/obj/item/gun/kinetic/single_action)
...
+/obj/item/gun/kinetic/foamdartshotgun
+	name = "foam dart shotgun"
+	icon_state = "foamdartshotgun"
+	desc = "The more powerful, older brother of the dart gun. Kicks like a horse, a foam horse. A horse made of foam."
+	inhand_image_icon = 'icons/mob/inhand/hand_guns.dmi'
+	item_state = "foamdartshotgun"
+	wear_state = "foamdartshotgun"
+	wear_image_icon = 'icons/mob/clothing/back.dmi'
+	contraband = 1
+	two_handed = 1
```
Comment (TobleroneSwordfish) [inline code/obj/item/gun/kinetic.dm:1046]:
```suggestion
	two_handed = TRUE
```

**Review by TobleroneSwordfish — APPROVED** [review]
Cute, more nerf battles!

**Comment by CheffieGithub** [comment]
I feel like having another static source that isn't loot crates would be good. you will NEVER see this thing otherwise. I suggest adding it to the firing range or securitron academy prefabs since I think those both have foam dart guns already.

**Comment by HatNJacket** [comment]
I'm unsure why changing the var from 1 to TRUE has caused checks to fail but I guess I'll revert the commit

**Comment by HatNJacket** [comment]
I can't figure out how to revert a commit on the website and I can't get onto the original machine for at least 5 hours to do it through vsc. Poor shotgun :(

**Comment by HatNJacket** [comment]
Apologies for requesting a second review, my brain is square and the fix was simple.

**Review by CheffieGithub — APPROVED** [review]
Yay for availability

**Pull request merged**